### PR TITLE
ISSD-1448 [PRM] Update Level dashboard's Target ARR amounts for Platinum partners

### PR DESCRIPTION
**Pull request by thaynam**

### CODE REVIEW

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/dashboard/LevelChart/components/PartnershipLevel/PartnershipLevel.tsx [hunk 1]
```diff
@@ -60,12 +60,12 @@ const PartnershipLevel = ({
...
-		if (partnerLevel.partnerLevelType.key === PartnershipLevels.GOLD) {
-			return partnerLevelProperties[partnerLevel.partnerLevelType.key]
-				.goalARR;
+		if (partnerLevel.partnerLevelType.key === PartnershipLevels.PLATINUM) {
+			return aRRResults.targetArr;
 		}
 
-		return aRRResults.aRRAmountTotal;
+		return partnerLevelProperties[partnerLevel.partnerLevelType.key]
+			.goalARR;
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/dashboard/LevelChart/components/PartnershipLevel/PartnershipLevel.tsx:68]: Melhor manter a logica original aqui.

```suggestion
		if (partnerLevel.partnerLevelType.key === PartnershipLevels.GOLD) {
			return partnerLevelProperties[partnerLevel.partnerLevelType.key]
				.goalARR;
		}

		return aRRResults.aRRAmountTotal;
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:forward:force

**Comment by thaynam** [comment]
ci:forward:force

**Comment by thaynam** [comment]
ci:forward:force

**Comment by Manda-vit** [comment]
ci:foward:force

**Comment by thaynam** [comment]
ci:forward:force

**Pull request closed without merging**